### DISCUSSION TITLE
why can't it connect to inav?

### DISCUSSION THREAD

**Discussion by Pacha240** [iNavFlight/inav-configurator#1586] [Q&A]
tmotor f7 hd flight controller. The firmware is being installed, but then I can’t connect to the drone and go into the settings. com4 is detected but does not enter. although it enters betafly, why can't it connect to inav?

**Comment by MrD-RC** [comment]
You really need ti give more information. Like: 
- what operating system and version are you using?
- what have you already tried to fix this?

**Reply by Pacha240** [reply]
drivers are installed. before that, I used betaflight and did not connect. wanted to go to inav, the firmware installs, but then does not get connected. when connecting the cable, it detects com4

**Reply by Pacha240** [reply]
windows 11
inav 5.0

**Reply by MrD-RC** [reply]
Windows 11 can be bad with the STM drivers. Click the link on the welcome page of configurator to download the STM32 drivers, and install them.
